tests/unit/account.js

```diff
@@ -133,14 +133,31 @@ describe('Unit/Account:', () => {
     describe('Signatures', () => {
         it('Success: Sign a message with BN signer', async () => {
             await expect(
-                madWallet.Account.accounts[1]["signer"].sign("0xc0ffeebabe")
+                madWallet.Account.accounts[0]["signer"].sign("0xc0ffee")
```

Comment:
Just staying consistent with other test hex strings

Reply:
Additionally updating to use the BN account instead of the SECP account added above in the file